draft-ietf-tls-rfc4492bis.xml

```diff
@@ -339,7 +340,10 @@
             deprecates all but the uncompressed point format. Implementations of this document MUST support the
             uncompressed format for all of their supported curves, and MUST NOT support other formats for curves
             defined in this specification. For backwards compatibility purposes, the point format list extension
-            MUST still be included, and contain exactly one value: the uncompressed point format (0).</t>
+            MAY still be included, and contain exactly one value: the uncompressed point format (0). RFC 4492 
+            specified that if this extension is missing, it means that only the uncompressed point format is 
+            supported, so interoperability with implementations that support the uncompressed format should work
+            with or without the extension.</t>
           <t> The ECPointFormat name space is maintained by IANA.  See <xref target="iana"/> for information on how
             new value assignments are added.</t>
           <t> Items in ec_point_format_list are ordered according to the client's preferences (favorite choice first).</t>
```

Comment:
If we do then we have to describe what to do if we get an extension advertising **only** compressed. That would mean we can't interoperate. So do we want to specify "bang your head against the wall and try" by ignoring this advertisement or evaluate and return some alert.